docs/UserGuide.md

```diff
@@ -242,7 +244,7 @@ Format: `sort -c INDEX`
 1. By lexicographical order of their email
 
 Examples:
-* `sort -c 2` would sort all currently displayed contacts by their names in alphabetical order
+* `sort -c 2` would sort all currently displayed contacts by their names in alphabetical order.
```

Comment:
This does not seem to follow the general example format of other commands. The format usually goes as 
"`{command}` Sorts all currently ... " starting with a capitalized verb